app/api.test.js

```diff
@@ -70,8 +79,8 @@ describe('api.js', ()=>{
     })
   })
 
-  describe('unpinObject', ()=>{
-    it('should reject when IPFS is not started', ()=>{
+  describe('unpinObject', () => {
```

Comment:
I can see our editors like to format differently :smile: , I will look into this at some point